### ISSUE TITLE
[Bug]: workflowengine / systemtag: Tags search still broken in 26 beta 5

### ISSUE THREAD

**Issue by ernolf** [nextcloud/server#36962]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Only the first 5 tags can be selected in the selection list that does not scroll

### Steps to reproduce

1. Setup a clean nextcloud 26 beta 5
2. Install workflow_script
3. go to /settings/admin and create at least 6 random tags
4. go to /settings/admin/workflow, "Ad new Flow" -> Filesystem Tag, Select Tag: only the first 5 Tags are shown and the rest can not be selected.

### Expected behavior

Complete tag-list available

### Installation method

Community Manual installation with Archive

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
:~# php -f occ config:list system
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "trusted_domains": [
            "192.168.188.40",
            "192.168.178.40"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "version": "26.0.0.7",
        "check_for_working_htaccess": false,
        "overwrite.cli.url": "http:\/\/192.168.188.40",
        "htaccess.RewriteBase": "\/",
        "dbtype": "mysql",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "default_phone_region": "DE",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "filelocking.enabled": true,
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0
        },
        "updater.release.channel": "beta",
        "maintenance": false,
        "theme": "",
        "log_query": false,
        "loglevel": 2,
    }
}
```

### List of activated Apps

```shell
z:~# php -f occ app:list
Enabled:
  - activity: 2.18.0
  - circles: 26.0.0-beta2
  - cloud_federation_api: 1.9.0
  - comments: 1.16.0
  - contactsinteraction: 1.7.0
  - dashboard: 7.6.0
  - dav: 1.25.0
  - federatedfilesharing: 1.16.0
  - federation: 1.16.0
  - files: 1.21.1
  - files_pdfviewer: 2.7.0
  - files_rightclick: 1.5.0
  - files_sharing: 1.18.0
  - files_trashbin: 1.16.0
  - files_versions: 1.19.1
  - firstrunwizard: 2.15.0
  - logreader: 2.11.0
  - lookup_server_connector: 1.14.0
  - nextcloud_announcements: 1.15.0
  - notifications: 2.14.0
  - oauth2: 1.14.0
  - password_policy: 1.16.0
  - photos: 2.2.0
  - privacy: 1.10.0
  - provisioning_api: 1.16.0
  - recommendations: 1.5.0
  - related_resources: 1.1.0-alpha1
  - serverinfo: 1.16.0
  - settings: 1.8.0
  - sharebymail: 1.16.0
  - support: 1.9.0
  - survey_client: 1.14.0
  - systemtags: 1.16.0
  - text: 3.7.1
  - theming: 2.1.1
  - twofactor_backupcodes: 1.15.0
  - updatenotification: 1.16.0
  - user_status: 1.6.0
  - viewer: 1.10.0
  - weather_status: 1.6.0
  - workflow_script: 1.11.1
  - workflowengine: 2.8.0
Disabled:
  - admin_audit: 1.16.0
  - bruteforcesettings: 2.6.0
  - encryption: 2.14.0
  - files_external: 1.18.0
  - suspicious_login: 4.4.0
  - twofactor_totp: 8.0.0-alpha.0
  - user_ldap: 1.16.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Additional info

I did a search for this issue and all searchresults finished with a linkt to nextcloud/nextcloud-vue/issues/3662 which leads to this (merged) pull request: nextcloud/nextcloud-vue/pull/3747 that has been integrated into Vue 7.6.0. : nextcloud/nextcloud-vue/pull/3755

However, since 26 beta 5 uses Vue 7.7.0 and the error STILL occurs, hence this issue raport.

**Comment by solracsf** [comment]
cc @Pytal

**Comment by Pytal** [comment]
Fixed in `v26.0.0rc1` with https://github.com/nextcloud/server/pull/36754
